testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -179,6 +179,9 @@ Feature: Synchronize products in the products page of the Setup Wizard
     Then I should not see a "Operation not successful" text
     And I should not see a warning nor an error sign
 
+  Scenario: Detect repository synchronization issues in reposync logs
```

Comment:
Let's please don't have it on CI, we can keep checking this on finishing.
Sometimes, we might have some error at thatog, but we might prefer to keep running the rest of CI stages

Reply:
Hmmm, we run CI more often than BV, it could be nice to catch things like  https://bugzilla.suse.com/show_bug.cgi?id=1228701